documentation/modules/con-creating-managing-connectors.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-connect.adoc
+
+[id='con-creating-managing-connectors-{context}']
+
+= Creating and managing connectors
+
+Once you have created a container image for your connector plug-in, you need to create a connector instance in your Kafka Connect cluster. You can then configure, monitor, and manage a running connector instance. For example, you can:
+
+* Check the status of a connector instance
+* Increase or decrease the number of tasks
+* Restart failed tasks
+* Pause a connector instance
+* Delete a connector instance
+
+{ProductName} provides two different APIs for creating and managing connectors:
+
+* `KafkaConnector` resources (referred to as `KafkaConnectors`)
+* Kafka Connect REST API
+
+For security reasons, we recommend that you use `KafkaConnectors` rather than the Kafka Connect REST API. `KafkaConnectors` use Kubernetes RBAC controls to secure the Kafka Connect REST API and lock down its resources.
```

Comment:
This isn't true and the messaging here is a little bit difficult. We want users to start using `KafkaConnectors` because we think that eventually we not expose the REST API at all (or at least not by default). But right now it doesn't support pause/resume or task restart, so it's not really feasible to lock it down.

Using NetworkPolicies they will be able to limit where requests to the Kafka Connect REST API come from. And using RBAC they will be able to control who has access the connectors. So the access control story is _potentially_ better using `KafkaConnector` than without. But it's not currently better by default.

Reply:
Thanks for the explanation. It's tricky alright. Rather than mention security at all, I suggest that we just have a single note.

NOTE: Currently, `KafkaConnectors` do not support restarting failed tasks and pausing and resuming connectors. You need to use the Kafka Connect REST API to perform these tasks.